test/ui/ViewJobRolesTests.ts

```diff
@@ -0,0 +1,62 @@
+import { ViewJobRolesTestsPage } from '../ui/ViewJobRolesTestsPage';
+import { HeaderTestPage } from '../ui/HeaderTestPage';
+import { FooterTestPage } from '../ui/FooterTestPage';
+import { basepage } from './basepage';
+
+describe('View Job Roles Tests', function () {
+    this.timeout(70000);
+    const page:string = 'https://jptw3amsi2.eu-west-1.awsapprunner.com/jobRoles';
```

Comment:
pass the URL in as an environment variable so we can run locally and against AWS